### DISCUSSION TITLE
Button

### DISCUSSION THREAD

**Discussion by bstro** [chakra-ui/zag#67] [Q&A] [answered]
Zag looks like a pretty sweet library and I'm thinking of switching out React-ARIA with Zag for my app. One thing for me that Zag.js is missing (understandably as the library was just released) is a button component that handles accessibility/keyboard navigation like React-ARIA.

https://react-spectrum.adobe.com/react-aria/useButton.html
https://react-spectrum.adobe.com/blog/building-a-button-part-1.html

**Comment by sifferhans** [comment]
This would be really nice to have! Buttons are relatively "simple" components, but accessibility is always hard to implement (to me, at least).

**Comment by sifferhans** [comment, marked as answer]
@bstro I guess the [pressable](https://zagjs.com/components/react/pressable) component might be what we're looking for 🤩

**Comment by sifferhans** [comment]
@bstro Could you mark this as answered if the `@zag-js/pressable` was the answer you were looking for?

**Reply by bstro** [reply]
I would, but the "mark as answered" button does not exist in my UI, likely because this was posted under "Ideas" and not "Q&A"

**Comment by anubra266** [comment]
Done @sifferhans @bstro
